cpp/src/arrow/c/bridge.cc

```diff
@@ -1977,6 +1978,24 @@ Result<std::shared_ptr<Array>> ImportDeviceArray(struct ArrowDeviceArray* array,
   return ImportDeviceArray(array, *maybe_type, mapper);
 }
 
+Result<std::shared_ptr<MemoryManager>> DefaultDeviceMapper(ArrowDeviceType device_type,
+                                                           int64_t device_id) {
+  if (device_type != ARROW_DEVICE_CPU) {
+    return Status::NotImplemented("Only importing data on CPU is supported");
+  }
```

Comment:
This could later be expanded to also allow CUDA device for CUDA enabled builds

Reply:
Yes, there probably should be some kind of registry so that device mappers can be added separately.

Reply:
@pitrou you mention a "registry", but AFAIK that's what we ideally would have (so external device implementations could register themselves) and that doesn't exist yet, right? 
In that case, is the function above an OK short-term default?

Reply:
Yes and yes!